src/widgets/infinite-hits/infinite-hits.tsx

```diff
@@ -216,7 +208,12 @@ const infiniteHits: InfiniteHits = (
     connectInfiniteHits
   )(specializedRenderer, () => render(null, containerNode));
 
-  return makeInfiniteHits({ escapeHTML, transformItems, showPrevious });
+  return makeInfiniteHits({
+    container: containerNode,
```

Comment:
You agree that this was a mistake not having `container` right?

Reply:
The containerNode is already passed to `specializedRenderer`, makeWidget is the connector, which doesn't use container I think. It does seem a little inconsistent with the other widgets though

Reply:
No, the connector doesn't need to be aware of the `container` (same as any other connector).

Reply:
@Haroenv @francoischalifour I see. Going to still pass to the container so we don't perform a TS ignore.

Reply:
I see. Going to pass the container anyways if sounds ok for you guys, because normally the connector returns a function that accepts the widgetParams. The widgetParams in the case requires a container.

Reply:
It means that there's an issue somewhere else then. Why did we not need to pass `container` in other connectors?

Connectors are not bound to any DOM, we use them for Vue InstantSearch, Angular InstantSearch and later, React InstantSearch, where DOM elements are not part of the API.

Reply:
Humm; let's zoom on this one?